scripts/dbsetup.py

```diff
@@ -10,7 +11,9 @@
         JobID INTEGER PRIMARY KEY AUTOINCREMENT,
         Comment TEXT,
         MediaPath TEXT,
-        OptionalText TEXT
+        OptionalText TEXT,
```

Comment:
could we like, make literally everything that's camelcase in this whole diff be snake case?